contracts/src/token/erc721/mod.rs

```diff
@@ -1853,40 +1862,40 @@ mod tests {
     }
 
     #[motsu::test]
-    fn owner_of_inner_works(contract: Erc721) {
+    fn _owner_of_works(contract: Erc721) {
```

Comment:
Why do we prefix these functions with an underscore?

Reply:
Pointless. I agree